src/call-session/index.ts

```diff
@@ -73,12 +85,29 @@ class CallSession extends EventEmitter {
     return this.localPeer ? extractNumber(this.localPeer) : "";
   }
 
+  public get remoteTag() {
+    return this.remotePeer ? extractTag(this.remotePeer) : "";
+  }
+
+  public get localTag() {
+    return this.localPeer ? extractTag(this.localPeer) : "";
+  }
+
   public get isConference() {
     return this.remotePeer
       ? extractNumber(this.remotePeer).startsWith("conf_")
       : false;
   }
 
+  public get rcHeaders() {
+    const rcHeaders = this.sipMessage?.headers["P-rc"];
+    if (!rcHeaders) {
+      return null;
+    }
+    const msg = RcMessage.fromXml(rcHeaders);
+    return msg.headers;
+  }
```

Comment:
If `rcHeader` is only used in one place (src/call-session/inbound.ts). Creating such a helper function won't make the whole solution any simpler

Reply:
There are sugar (getter) methods provided:  `callSession.sessionId` and `callSession.partyId`.